apps/researcher/src/lib/community/clerk-converters.test.ts

```diff
@@ -0,0 +1,193 @@
+import {describe, expect, it} from '@jest/globals';
+import {Organization, OrganizationMembership} from '@clerk/backend/dist/types';
+import {
+  ClerkPaginatedResponse,
+  GetMembersParams,
+  GetMembershipsParams,
+  OrganizationDomainResource,
+  OrganizationInvitationResource,
+  OrganizationMembershipRequestResource,
+  OrganizationMembershipResource,
+  OrganizationResource,
+} from '@clerk/types';
+import {Community, Membership} from './definitions';
+import {
+  organizationMembershipToCommunityMembership,
+  organizationToCommunity,
+} from './clerk-converters';
+
+const organization: Organization = {
+  id: 'org1',
+  name: 'Organization 1',
+  publicMetadata: {
+    description: 'Description',
+    attributionId: 'attributionId',
+    licence: 'licence',
+  },
+  slug: 'organization-1',
+  imageUrl: 'https://example.com/image.png',
+  createdAt: 1620000000000,
+  updatedAt: 1620000000000,
+  members_count: 10,
+  logoUrl: 'https://example.com/logo.png',
+  hasImage: true,
+  createdBy: 'user1',
+  privateMetadata: {},
+  maxAllowedMemberships: 100,
+  adminDeleteEnabled: true,
+};
+
+const expectedCommunity: Community = {
+  id: 'org1',
+  name: 'Organization 1',
+  description: 'Description',
+  attributionId: 'attributionId',
+  licence: 'licence',
+  slug: 'organization-1',
+  imageUrl: 'https://example.com/image.png',
+  createdAt: 1620000000000,
+  membershipCount: 10,
+  canAddEnrichments: true,
+};
+
+describe('organizationToCommunity', () => {
+  it('converts an Organization to a Community', () => {
+    expect(organizationToCommunity(organization)).toEqual(expectedCommunity);
+  });
+
+  it('converts an OrganizationResource to a Community', () => {
+    const organization: OrganizationResource = {
```

Comment:
For readability: do we need to implement all props and functions of an `OrganizationResource` here or can we only provide the props that are relevant for the `organizationToCommunity` conversion?

Reply:
I would love to do this, but without all the props, it is not a valid `OrganizationResource`.

Reply:
I understand. For the unit test, though, it doesn't seem to matter - `organizationToCommunity` only needs specific props from `OrganizationResource`? If so, you could silence TS by setting a `// @ts-expect-error:TS{error code}`

Reply:
Yes, this will work, thank you.